code/modules/research/designs/autolathe_designs.dm

```diff
@@ -539,6 +539,14 @@
 	build_path = /obj/item/device/radio/electropack
 	category = list("hacked", "Tools")
 
+/datum/design/foam_dart
+	name = "Box of Foam Darts"
+	id = "foam_dart"
+	build_type = AUTOLATHE
+	materials = list("$metal" = 500)
+	build_path = /obj/item/ammo_box/foambox
+	category = list("initial", "Misc")
```

Comment:
this won't show up, as we don't have a "Misc" category. You'll want full out `"Miscellaneous"`